packages/tools/src/commands/create-plugin/index.ts

```diff
@@ -0,0 +1,120 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import checkbox from '@inquirer/checkbox';
+import input from '@inquirer/input';
+import { Command, Option } from 'clipanion';
+import fs from 'fs-extra';
+import path from 'path';
+
+import { ROOT } from '../../constants';
+import { green, slugify, runAutoFixes } from '../../helpers';
+import type { Context, Plugin } from '../../types';
+import { updateFiles } from '../integrity/files';
+import { updateReadmes } from '../integrity/readme';
+
+import { getFiles } from './templates';
```

Comment:
Avoid top level `import`s as you're using a require on each file for `yarn cli`

so all those imports are slowing down the other commands (even if you're not running them)

Reply:
Oh yeah I wanted to move them in a separate file, I'll do it.